### PR TITLE
THU/add_scenarios_pbi_filter

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/views/Dashboards/Dashboards.js [hunk 1]
```diff
@@ -82,8 +90,17 @@ const Dashboards = () => {
         </Card>
       </Grid>
       <Grid item sm={10} className={classes.dashboardsMainContainer}>
-        <Card className={classes.dashboard}>
-          <CardContent>{<TabPanel index={value} key={dashboardTitle} title={dashboardTitle} />}</CardContent>
+        <Card className={classes.dashboardsMainCard}>
+          <CardContent className={classes.dashboardsMainCardContent}>
+            {
```
Comment (ardvi) [inline src/views/Dashboards/Dashboards.js:95]: I think that brackets are useless here

**Review by ardvi — APPROVED** [review]
LGTM , just a little comment

File: src/views/Dashboards/Dashboards.js (hunk 1 above)
Comment (csm-thu) [reply src/views/Dashboards/Dashboards.js]: Fixed it, thank you :slightly_smiling_face:

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request merged**